source/switching_to_live/set_up_a_live_stripe_account/index.html.md.erb

```diff
@@ -1,17 +1,17 @@
 ---
-title: Connect your live account to Stripe
-last_reviewed_on: 2025-03-26
+title: Connect your live service to Stripe
+last_reviewed_on: 2025-11-13
 review_in: 6 months
 weight: 7110
 ---
 
-# Connect your live account to Stripe
+# Connect your live service to Stripe
 
-This is part of [going live](/switching_to_live/#go-live). You need to complete the steps on that page before continuing here.
+This is part of [going live](/switching_to_live/#go-live). You must complete the steps on that page before continuing here.
 
 Stripe is GOV.UK Pay's contracted PSP.
 
-After you request a live GOV.UK Pay account and confirm you will connect it to Stripe, we'll review your request to go live. After we approve your request, you need to add more details about your service and organisation in the GOV.UK Pay admin tool.
+After you request a live GOV.UK Pay service, we'll review your request to go live. After we approve your request, you need to add more details about your service and organisation in the GOV.UK Pay admin tool.
```

Comment:
Might "request a live GOV.UK Pay service" be confusing, as even to users it's the same service, just being put into live mode?